gmm/handler.go

```diff
@@ -1622,13 +1623,14 @@ func HandleUeSliceInfoDelete(ue *context.AmfUe, accessType models.AccessType, ns
 
 	// we are doing local cleanup for now, below code will be deprecated when we support
 	// Configuration Update
+	var problemDetail *models.ProblemDetails
```

Comment:
To be aligned to the previous change, should not this be `problemDetails`?
```suggestion
	var problemDetails *models.ProblemDetails
```

Reply:
Done